test/testkit/k8s/network_policy.go

```diff
@@ -0,0 +1,35 @@
+package k8s
+
+import (
+	networkingv1 "k8s.io/api/networking/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type NetworkPolicy struct {
```

Comment:
Let's make this name more specific. Maybe `DenyAllNetworkPolicy`?

Reply:
done